webapi/webapi-sensors-w3c-tests/sensors/Accelerometer.https.html

```diff
@@ -47,6 +47,24 @@
   return new Array(reading.x, reading.y, reading.z);
 }
 
+async_test(t => {
+  let sensor1 = new Accelerometer();
+  let sensor2 = new Accelerometer({includeGravity: false});
+  sensor1.start();
+  sensor2.start();
+  sensor1.onchange = t.step_func_done(() => {
+    let cached1 = arr_sensor_reading(sensor1.reading);
+    let cached2 = arr_sensor_reading(sensor2.reading);
+    for (var i = 0; i < cached2.length; i++) {
+      assert_not_equals(cached1[i], cached2[i]);
+    }
+  });
+  sensor1.onerror = t.step_func_done(event => {
+    assert_unreached(event.error.name + ":" + event.error.message);
+  });
+}, "Test that get different X, Y and Z axis values by includeGravity are true and false.");
```

Comment:
Test that when construct Accelerometer with includeGravity that is set to false, the reading is different from the one with default Accelerometer.